news/_posts/2023-10-10-release-20231010.md

```diff
@@ -0,0 +1,43 @@
+
+All the aforementioned games are standalone, though non-original games may require you to own the copy-rightted assets. Some of these games choose to contribute back to the OpenRA project, and we greatly appreciate all the help we can get.
+
+There are also many promising OpenRA based games in progress, including [OpenE2140](https://www.moddb.com/mods/opene2140), a remake of the old Earth 2140. A monster game [Cameo](https://www.moddb.com/mods/cameo) featuring over 200 playable factions (subfactions included in the count) from a massive selection of games.
+
+Our competitive community is going strong as well. [Red Alert Global League 15th season](https://forum.openra.net/viewtopic.php?f=85&t=21802) has just started, and for Halloweeen you can expect [Dark Tournament VII](https://forum.openra.net/viewtopic.php?f=85&t=21787), registrations are currently open. [RAGL's 14th season](https://forum.openra.net/viewtopic.php?f=85&t=21739) has wrapped up about 4 months ago, and for its coverage you can chekout [Weekly Highlight forum posts](https://forum.openra.net/viewtopic.php?f=85&t=21755), casted by [Kaution](https://www.youtube.com/@kaution6829) and [here is the link to the playlist](https://www.youtube.com/watch?v=Osa6aH5Y75I&list=PLHuw_QR21dC_kKaYagJR-KCgZH2s_KHaz). There are also plenty of other creators covering the tournament: [FiveAces Shoutcasts](https://www.youtube.com/@CovertFlobert), [Blackened's video reviews](https://www.youtube.com/@Blackeneds_Barracks) or [Milkman's streams](https://www.twitch.tv/mm_openra/videos).
+
+<figure>
+  <img src="{{ '/images/news/20231010-darktournament.jpg' | relative_url }}" style="width: 800px" alt="Dark Tournament VII" />
+</figure>
+
+And as last time, we would like mention that this release contains significant progress towards C&C Remastered Collection support, including compatibility with the new EA app. The "Tiberian Dawn HD" project is a separate release and can be [downloaded here](https://github.com/OpenRA/TiberianDawnHD/releases/tag/release-20231010). This preview is multiplayer-compatible with the main 20231010 release. However, please keep in mind that performance, memory usage and loading times have not been fully optimized yet. The C&C Remastered Collection must be installed through Steam or the EA App, and in case you are using macOS or Linux the project README provides you [detailed installation instructions](https://github.com/OpenRA/TiberianDawnHD#asset-installation).
```

Comment:
```suggestion
And as last time, we would like to mention that this release contains significant progress towards C&C Remastered Collection support, including compatibility with the new EA app. The "Tiberian Dawn HD" project is a separate release and can be [downloaded here](https://github.com/OpenRA/TiberianDawnHD/releases/tag/release-20231010). This preview is multiplayer-compatible with the main 20231010 release. However, please keep in mind that performance, memory usage and loading times have not been fully optimized yet. The C&C Remastered Collection must be installed through Steam or the EA App, and in case you are using macOS or Linux the project README provides you [detailed installation instructions](https://github.com/OpenRA/TiberianDawnHD#asset-installation).
```